examples/example_matching.py

```diff
@@ -37,7 +40,16 @@
 p_ch = "./example_data/companies_house_addresess_postcode_overlap.parquet"
 p_fhrs = "./example_data/fhrs_addresses_sample.parquet"
 
-con = duckdb.connect(database=":memory:")
+con = duckdb.connect(
+    database=":memory:",
+    config={
+        "allow_unsigned_extensions": "true",
+        "allow_community_extensions": "true",
+    },
+)
+
+con.execute("FORCE INSTALL splink_udfs FROM community;")
```

Comment:
just for info, the `FORCE` here prevents it using the cache e.g. if you've just done a new release and want to make sure you get the latest code